packages/beacon-node/test/e2e/eth1/jsonRpcHttpClient.test.ts

```diff
@@ -151,19 +147,24 @@ describe("eth1 / jsonRpcHttpClient", function () {
       const controller = new AbortController();
       if (abort) setTimeout(() => controller.abort(), 50);
       const eth1JsonRpcClient = new JsonRpcHttpClient([url], {signal: controller.signal});
-      await expect(eth1JsonRpcClient.fetch(payload, {timeout})).to.be.rejected.then((error) => {
+
+      try {
+        await eth1JsonRpcClient.fetch(payload, {timeout});
+      } catch (error) {
         if (testCase.errorCode) {
-          expect((error as FetchError).code).to.be.equal(testCase.errorCode);
+          expect((error as FetchError).code).toBe(testCase.errorCode);
         } else {
-          expect((error as Error).message).to.include(testCase.error);
+          expect((error as Error).message).toEqual(expect.stringContaining(testCase.error));
         }
-      });
+      }
+      expect.assertions(1);
     });
   }
-});
+}, {timeout: 10_000});
 
+// To make the code review easy for code block below
+/* prettier-ignore */
```

Comment:
does this mean this line should be removed after review?

Reply:
Yes my suggestion would be to remove these afterwards.